xml/entity-decl.ent

```diff
@@ -10,7 +10,7 @@
 <!ENTITY latestuaachart "2.15.2">
 <!ENTITY latestscfchart "2.15.2">
 <!ENTITY lateststratoschart "2.3.0">
-<!ENTITY aksnodeversion "1.11.8">
+<!ENTITY aksnodeversion "1.13.5">
```

Comment:
See other comment above about versioning. If 1.11.8 is no longer available we should change this to the version being tested by our team.